addons/spreadsheet/static/src/scss/spreadsheet_dashboard.scss

```diff
@@ -0,0 +1,3 @@
+.o-dashboard-opacity-25 .o-figure {
```

Comment:
I would name this with some semantics: o-dashboard-sample-opacity or o-dashboard-placeholder-opacity